package.json

```diff
@@ -3,40 +3,40 @@
   "private": true,
   "version": "0.15.0-alpha",
   "devDependencies": {
-    "babel": "^5.8.3",
+    "babel": "^5.8.29",
     "babel-eslint": "4.1.3",
     "benchmark": "^1.0.0",
-    "browserify": "^9.0.3",
+    "browserify": "^12.0.0",
     "bundle-collapser": "^1.1.1",
     "coffee-script": "^1.8.0",
-    "del": "^1.2.0",
-    "derequire": "^2.0.0",
+    "del": "^2.0.2",
```

Comment:
If we switched over entirely to gulp today this would actually cause issues. It won't right now because the only place that uses it will never get used in a sequence (the major version number change here was actually important). Let's leave this.

The rest looks like it should be fine. We'll make other major changes as one-offs (eg, switching to babel@6)